docs/StardustDocs/topics/split.md

```diff
@@ -131,18 +135,7 @@ df.split { "name"["lastName"]<String>() }
 <dataFrame src="org.jetbrains.kotlinx.dataframe.samples.api.Modify.split1.html"/>
 <!---END-->
 
-`String` columns can also be split into group matches of [`Regex`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/-regex/) pattern:
-
-<!---FUN splitRegex-->
-
-```kotlin
```

Comment:
What's happened with this example? What's its future here and in codebase?

Reply:
Like I described here https://github.com/Kotlin/dataframe/issues/562, the example is not incorrect, but it show a bit too much information before getting to the point. It first merges string columns before splitting them again, while only the splitting part is relevant here. It should still exist in the code base to generate the right DFs tho.